I.write-tests/06-largest-number/largest-number.test.js

```diff
@@ -1,3 +1,16 @@
+var getLargestNumber = require('./largest-number');
+
+test('Get largest number',function (){
+    const array = [3, 21, 88, 4, 36];
+    const expectation = 88;
+    const result = getLargestNumber(array);
+  
+
```

Comment:
this test doesn't seem to be making any assertions

Reply:
I forgot it . I wondering why the test pass without it?